services/src/layers/add_from_directory.rs

```diff
@@ -92,6 +117,19 @@ pub async fn add_layer_collections_from_directory<L: LayerDb>(db: &mut L, file_p
         )
         .await?;
 
+        // share with users
+        debug!("sharing collection");
```

Comment:
add more infos (which collection) or remove debug

Reply:
I just moved the code. I don't know who put it here and why.

Reply:
[bcb0f9d](https://github.com/geo-engine/geoengine/pull/1008/commits/bcb0f9d958e515b6137e5ecfb81547892e713ba6)